### ISSUE TITLE
NTR: Conjunctival abnormal pigmentation (Eye structure & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7983]
New term request Conjunctival abnormal pigmentation (elthat_110418233719):   UMLS: 0155163

**Comment by pnrobinson** [comment]
@MickeySegal
could we specify what kind of abnormal pigmentation is meant? What is the context_

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK1454/
"Brown pigmentation of the sclera is observed midway between the cornea and the outer and inner canthi at the insertion of the recti muscles. Pigment deposition may also be seen in the conjunctiva and cornea"

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3251793/
"hyperpigmentation of conjunctiva"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000365
- label: Brown pigmentation of the conjunctiva
- parents: [Abnormal conjunctiva morphology (HP:0000502)](https://hpo.jax.org/app/browse/term/HP:0000502)
- pmid: PMID:15249376|PMID:20301627
- def: Deposition of brown pigmentation in the conjunctiva.
- comment: This feature is a manifestation of alkaptonuria.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7983
- synonyms:
